packages/client/hmi-client/src/views/Home.vue

```diff
@@ -4,65 +4,54 @@ import NewProjectCard from '@/components/projects/NewProjectCard.vue';
 import IconTime32 from '@carbon/icons-vue/es/time/32';
 import IconChevronLeft32 from '@carbon/icons-vue/es/chevron--left/32';
 import IconChevronRight32 from '@carbon/icons-vue/es/chevron--right/32';
-import IconFire32 from '@carbon/icons-vue/es/fire/32';
-import IconCoronavirus32 from '@carbon/icons-vue/es/coronavirus/32';
 
 const enum Categories {
 	Recents = 'Recents',
 	Trending = 'Trending',
 	Epidemiology = 'Epidemiology'
 }
 
-const categories = new Map<string, { icon: object }>([
-	[Categories.Recents, { icon: IconTime32 }],
-	[Categories.Trending, { icon: IconFire32 }],
-	[Categories.Epidemiology, { icon: IconCoronavirus32 }]
-]);
+const categories = new Map<string, { icon: object }>([[Categories.Recents, { icon: IconTime32 }]]);
 
-const dummyProjects: string[] = Array(8)
-	.fill('Project')
-	.map((project, index) => `${project} ${index}`);
+const mockProjects: string[] = ['Mocked Project'];
 </script>
 
 <template>
 	<section>
 		<h2>Projects</h2>
 		<template v-for="[key, value] in categories" :key="key">
-			<template v-if="dummyProjects.length > 0 || key === Categories.Recents">
-				<header>
-					<component :is="value.icon" />
-					<h3>{{ key }}</h3>
-				</header>
-				<div class="project-carousel">
-					<IconChevronLeft32 class="chevron-left" />
-					<ul>
-						<li v-if="key === Categories.Recents">
-							<NewProjectCard />
-						</li>
-						<li v-for="(project, index) in dummyProjects" :key="index">
-							<ProjectCard :name="project" />
-						</li>
-					</ul>
-					<IconChevronRight32 class="chevron-right" />
-				</div>
-			</template>
+			<header>
+				<component :is="value.icon" />
+				<h3>{{ key }}</h3>
+			</header>
+			<div class="project-carousel">
+				<IconChevronLeft32 class="chevron-left" />
+				<ul>
+					<li v-if="key === Categories.Recents">
+						<NewProjectCard />
+					</li>
+					<li v-for="(project, index) in mockProjects" :key="index">
+						<ProjectCard :name="project" />
+					</li>
+				</ul>
+				<IconChevronRight32 class="chevron-right" />
+			</div>
 		</template>
 	</section>
 </template>
 
 <style scoped>
 section {
+	background-color: var(--un-color-body-surface-secondary);
```

Comment:
We are not.

I do it because I find it easier to look for a property and it's aesthetically pleasing.

Reorganizing the CSS properties alphabetically of a component is my warm-up process before doing my actual work. But I should be careful sometimes I can go overboard and it makes it harder to review my PRs.